src/microbe_stage/systems/EndosymbiontOrganelleSystem.cs

```diff
@@ -74,7 +75,7 @@ protected override void Update(float state, in Entity entity)
                     // times and when not empty mostly just once
                     organelleContainer.OnOrganellesChanged(ref entity.Get<CompoundStorage>(),
                         ref entity.Get<BioProcesses>(), ref entity.Get<Engulfer>(), ref entity.Get<Engulfable>(),
-                        ref entity.Get<CellProperties>());
+                        ref entity.Get<CellProperties>(), tempProcessListMemory);
```

Comment:
The list must be locked as this can run multithreaded. See for example line 100 how the hexWorkData has to be locked.